crates/gpui/src/platform/mac/events.rs

```diff
@@ -121,6 +121,11 @@ impl PlatformInput {
                 NSEventType::NSFlagsChanged => {
                     Some(Self::ModifiersChanged(ModifiersChangedEvent {
                         modifiers: read_modifiers(native_event),
+                        capslock: Capslock {
+                            on: native_event
+                                .modifierFlags()
+                                .contains(NSEventModifierFlags::NSAlphaShiftKeyMask),
```

Comment:
This should probably be `NSEventModifierFlagCapsLock`

Reply:
I tested the code on my macOS (15.4.1) for `NSAlphaShiftKeyMask` the functionality works as expected. I also could not find `NSEventModifierFlagCapsLock` in cocoa crate we use in Zed. I saw it in other crates that they do have `NSEventModifierFlagCapsLock` but and it is the same value as `NSAlphaShiftKeyMask`

Reply:
Lovely! I think that’s fine then.